scripts/events_module/event_filters.py

```diff
@@ -406,6 +406,8 @@ def cat_for_event(constraint_dict: dict, possible_cats: list, comparison_cat=Non
     # run funcs
     allowed_cats = []
     for param in func_dict:
+        if not constraint_dict.get(param):
```

Comment:
```suggestion
        if param not in constraint_dict:
```

This is a bit more conventional.